topology/src/main/java/io/camunda/zeebe/topology/ClusterTopologyManagerImpl.java

```diff
@@ -245,10 +245,12 @@ private void applyTopologyChangeOperation(final ClusterTopology mergedTopology)
       return;
     }
 
+    final var updatedTopology = initialized.get();
```

Comment:
:wrench: Can we call this `initializedTopology`? `updatedTopology` sounds to me like the result of applying the transformer which happens later.